packages/editor/src/components/post-author/index.js

```diff
@@ -1,71 +1,137 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
 /**
  * WordPress dependencies
  */
+import { useState, useMemo } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import { withInstanceId, compose } from '@wordpress/compose';
-import { Component } from '@wordpress/element';
-import { withSelect, withDispatch } from '@wordpress/data';
-import { decodeEntities } from '@wordpress/html-entities';
+import { ComboboxControl } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import PostAuthorCheck from './check';
 
-export class PostAuthor extends Component {
-	constructor() {
-		super( ...arguments );
+let latestAvailableAuthors;
```

Comment:
This won't work when the component is used more than once or when a concurrent mode is enabled.

Reply:
Good point. This was added as a way to capture the available authors  so we can display them greyed out during a new data load (after typing more into the search). I'll remove this for now and we can work to address the UI issues separately.

Reply:
removed this, added an initial spinner instead to display while loading new results.